packages/components/src/components/action/action.scss

```diff
@@ -215,33 +208,35 @@
 :host([scale="s"]) {
   .button {
     @apply text-n2h font-normal;
-    padding-inline: var(--calcite-internal-action-padding-inline, theme("spacing.2"));
+    min-block-size: 24px;
```

Comment:
This should use a spacing token. Maybe `--calcite-size-fixed-xxl` or `--calcite-size-sm`? Applies to other `min-block-size` assignments.

Reply:
@matgalla